### ISSUE TITLE
Lipiduria

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4857]
**Preferred term label:**

**Synonyms**
Increased lipid levels in the urine

**Definition (free text, please give PubMed ID)**
An increase of lipid content in the urine.
PMID: 26124059
**Parent term (use hpo.jax.org/app)**
Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 301500
FABRY DISEASE

**Your nano-attribution (ORCID)**

0000-0001-9969-8610

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC292893/
and
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5987241/ (Fabry)

**Comment by pnrobinson** [comment]
exact synonym urinary lipid excretion

**Comment by pnrobinson** [comment]
Added as HP:0032567
